README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=main)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=main) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue) [![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![StackOverflow](https://img.shields.io/badge/StackOverflow-questions-blueviolet)](https://stackoverflow.com/questions/tagged/fairlearn)
+[![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=main)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=main) [![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/fairlearn/fairlearn/blob/main/LICENSE) [![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)](https://badge.fury.io/py/fairlearn) [![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![StackOverflow](https://img.shields.io/badge/StackOverflow-questions-blueviolet)](https://stackoverflow.com/questions/tagged/fairlearn)
```

Comment:
does this have to be a single line?

Reply:
I believe so. Otherwise they're on several lines. One day we'll be as fancy as scikit-learn and convert it into a README.rst where we have more flexibility 🙂

Reply:
I see, ok.

Reply:
Thanks for opening #884, I'll wait for that to be merged and make the changes in the new README.rst 🙂

Reply:
This is now much easier to read because we migrated to README.rst, so I'll resolve this comment.